roles/openshift_metrics/tasks/uninstall_metrics.yaml

```diff
@@ -18,3 +18,14 @@
     clusterrolebinding/heapster-cluster-reader
     clusterrolebinding/hawkular-metrics
   changed_when: delete_metrics.stdout != 'No resources found'
+
+# Update asset config in openshift-web-console namespace
+- name: Remove metrics route information from web console asset config
```

Comment:
I don't like this pattern.

Metrics, logging, whatever else should just call the standard webconsole role, that role's configs should be driven by inventory variables, not via vars from tasks/plays.

Reply:
When I suggested this to Sam we were trying to come up with a model where the implementation details were in openshift_web_console but driven by data/state changes in roles that need to update console. These state changes wouldn't necessarily always have corresponding inventory variables and we wanted to leave the logic for maintaining the values in the logging and metrics roles. Can we wait until we decide on where to centralize values like the metrics url to revisit this?